docs/src/content/docs/zh-cn/resources/showcase.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Starlight Showcase
```

Comment:
Was this not translated on purpose? Or maybe just forgot?

Reply:
Yeah. I updated it.